CHANGELOG.md

```diff
@@ -50,6 +62,22 @@
      command. This addition makes it possible to pipe commands like
      `packer plugins installed` with it for speedy cleanup of installed plugins.
      [GH-12886](https://github.com/hashicorp/packer/pull/12886)
+* core: Relax Packer source address URIs within the `required_plugins` block to
+     support the installation of local plugin binary using a custom or internal
+     source address (e.g. mycompay.com/plugins/happyorg/happycloud). Remote
+     installation using `packer init` or `packer plugins install`  does not
+     support non-GitHub source URIs. Users using internal source URIs must
```

Comment:
Just to make it clear that any non-Github URI can use this method, not just internal hosts

```suggestion
     support non-GitHub source URIs. Users using alternative hosts must
```